pkg/media/params.go

```diff
@@ -129,7 +137,17 @@ func isNilVideoParams(options *livekit.IngressVideoOptions) bool {
 		return true
 	}
 
-	if len(options.Layers) == 0 {
+	if _, ok := options.GetEncodingOptions().(*livekit.IngressVideoOptions_Preset); ok {
+		logger.Infow("presets unimplemented")
```

Comment:
nit: this might be a bit noisy

Reply:
This is meant to be temporary until I actually implement presets. Right now, presets are defined in the protocol, but you can't use them.

Reply:
Would failing the ingress instead match the caller's expectations better?

Reply:
yeah, for now I'd return an `unimplemented` error